modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
```

Comment:
This module does not appear to be included in the ocp-4-15-release-notes assembly. Adding an orphan module to the repository is to be avoided if at all possible. As the same information appears to be added to the assembly itself, I would suggest removing this file.

Reply:
Release notes are weird. I see why you've included this @obrown1205 but @mramendi is correct this just needs to be deleted.

Reply:
Done :)